airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml

```diff
@@ -16,16 +16,9 @@ tests:
   discovery:
     - config_path: "secrets/config.json"
       backward_compatibility_tests_config:
-        disable_for_version: "0.1.83"
+        disable_for_version: "0.2.2"
```

Comment:
please include a comment indicating why it was disabled for this version, to give the next developer an understanding of why it was done

Reply:
should this be a major upgrade if it's not backward compatible?

Reply:
I thought it should. Could you please clarify it?